information/Keys.md

```diff
@@ -0,0 +1,38 @@
+## Introduction
+
+* What are keys?
+
+* Why are (encryption) keys important in this industry?
+
+Meeco’s encryption key management service enables users to generate, manage and deploy keys to encrypt and protect data at rest, in transit and when exchanged. Our data and key architecture is designed for non-correlation, with all attributes encrypted with a Universally Unique Identifier (UUID). Provisioning UUIDs provides the foundation for Web3 applications where business rules and value drivers can be aligned to single identifiers.
+
+## Overview ##
+
+With a strong focus on privacy and security around the management and exchange of data, Meeco understands the importance of data encryption. With this in mind, we provide a key management service called the Keystore that enables users to easily access their data whilst ensuring it remains secure and protected. To do this we employ multiple cryptographic key types, all of which serve a purpose:
```

Comment:
Let's not introduce "Keystore" as a term.

Reply:
Renamed everywhere "Keystore" mention to KMS (Key management service